src/lplib/cmap.ml

```diff
@@ -0,0 +1,162 @@
+(*Type of points and intervals. Used to determine if a cursor is in the range
+of a specific token, which is an interval of points (positions)*)
```

Comment:
Thanks to add a space after `(*` and end your sentences with a dot. ;-)

Reply:
Most of this solved.